tests/format/flow/as-const/jsfmt.spec.js

```diff
@@ -0,0 +1,5 @@
+run_spec(import.meta, ["flow"], {
+  errors: {
+    "babel-flow": ["embedded.js"],
```

Comment:
Babel cannot parse `as const`, I'll report it later.

Reply:
Newly added flow syntax all not supportted. It's fine.